_config.yml

```diff
@@ -61,7 +61,7 @@ footer-link-col: "#404040"
 # Alternatively, the navbar, footer, and page background can be set to use background images
 # instead of colour
 
-# navbar-img: "/img/bgimage.png"
+navbar-img: "../img/rse-transport-light.png"
```

Comment:
Should we use the same background image in the footer, for consistency?

Reply:
Or at least, the same color changes?

Reply:
I didn’t give it much thought! Let me try it out quickly and see. Do you think a bold footer that sandwiches the page is better than a subtle one that doesn’t distract from the header and content?

Reply:
My thought was that headers and footers tend to be symmetric, to sandwich the page as you say.

Reply:
They can be, you can see everything from an off gray to black, to the exact same image. Here it is, as it is now (keeping a gray):

![image](https://user-images.githubusercontent.com/814322/59628735-5817c300-910f-11e9-8501-e5fa1853842c.png)

and here it is if we set the same image:

![image](https://user-images.githubusercontent.com/814322/59628790-6e258380-910f-11e9-82fe-3785cccea19c.png)

of course we'd need to fix the text color!

I actually like the dark, but having the double purple is a bit much for color (it's generally good to be selective about colors). We could also mirror something that the UK rse site did? Here is their footer:

![image](https://user-images.githubusercontent.com/814322/59628952-c066a480-910f-11e9-88cc-a82bbaa1566b.png)

So here is an example with ours:

![image](https://user-images.githubusercontent.com/814322/59629117-23583b80-9110-11e9-95e0-78ae6a586ec5.png)

I personally am of the mindset of having simplicity, and being careful about the choice of bright and dark colors. In that the footer was thin, I think I'd go for a dark (non purple) one like above. But it's fairly large, so instead it would draw attention away from what's above it. I'd definitely like to hear others' opinions too - I'm partial to the light colors, but could see the black with purple line being a reasonable option. I don't like the double purple - it's just too much purple.

Reply:
Now that I see the examples, I agree that there is too much purple in the double-purple option.  But I do still like the idea of symmetry between the top and bottom - it helps the reader parse what material is where.

Maybe a medium-brightness color would work, say a medium maroon or a medium gray?  Then you'd still have the symmetric colors but not draw attention away from the main section as much.

Reply:
Medium maroon... is this too light? Can you give me a hex code with what you are thinking?

![image](https://user-images.githubusercontent.com/814322/59630032-47b51780-9112-11e9-8789-ec1652c6ce9e.png)

Reply:
That one is #e0d2db which I grabbed from a pixel in the logo.

Reply:
This is a hard color because if you go brighter it gets very pink / purple (in a bad way) and if the maroon quickly can become dark pink.

Reply:
I was thinking something along the lines of #9e2871 for the maroon, or #bcb8b3 for the gray.  (These are both taken from the UK logo; I've been trying to reuse their colors whenever it makes sense, to help with co-branding.)

Reply:
Let's test them out! First #9e2871. I really like this color, but it might be too much for some:

![image](https://user-images.githubusercontent.com/814322/59630641-da09eb00-9113-11e9-8113-6441b1bcaa8e.png)

(it's very pink!)

and here is the grey:

![image](https://user-images.githubusercontent.com/814322/59630705-01f94e80-9114-11e9-944b-c1d4cc00087e.png)

Reply:
I think I still prefer the lighter grey, but we can see what others think.

Reply:
I really like dark footers, but usually in the case that they are contrast to a white (or very light) header. Here is a nice example (Parabricks) of such a combination:

![image](https://user-images.githubusercontent.com/814322/59630863-66b4a900-9114-11e9-90b0-1d635f2e6c3d.png)

Reply:
Yikes - that pink is very pink!  Funny how these colors look different than you imagined when you try them out on the actual site.

I think I prefer the medium gray, but I could go with any of the gray/black choices.

Reply:
Haha yea, it's exactly the same when you are painting a room.

Reply:
If you want to ping some others to bring in more feedback, I'm good with majority rules :)

Reply:
I have what might be an unhelpful suggestion/comment.  When it gets to color selection beyond something pretty basic, I get a little worried about UX and accessibility.  I'm *not* even remotely an expert on this, but I have some colleagues who have sufficiently drilled into me the importance of addressing these things.  I know there are places to check color contrasts to make sure they are deemed acceptable by whatever the accepted standards are.   Like this: https://webaim.org/resources/contrastchecker/ (first one google told me).

Reply:
That said, I think I would have a slight preference for the pink header, grey footer.  But what about a thin line of the pink at the top of the footer?

Reply:
@cosden I can try this out - which shade of gray are you referring to, and when you say "thin line of pink" do you mean a color (dark maroonish) to match the header?

Reply:
Yes, the dark maroonish - sorry! I thought you two were calling it pink.  I'm all confused.  I was thinking the light grey.  But again, I don't have a strong preference.

Reply:
okay testing this now! No worries, the thing is, if you take some of the pixels in the logo (that look purple) it totally comes out to Barbie doll high heel pink. So it's good to check :)

Reply:
okay here are "lighter grey with purple/pink border on top" at varying levels of thickness

5px

![image](https://user-images.githubusercontent.com/814322/59682939-39fca200-91a5-11e9-8388-3373e6d62c6a.png)

20px

![image](https://user-images.githubusercontent.com/814322/59683043-64e6f600-91a5-11e9-822a-50eb44c3dc17.png)

I still say too much purple. @cosden could you describe the accessibility issues you are worried about? Maybe that would be a good place to start our testing.

Reply:
If we had a lot of good organized links down there, I'd say yes, definitely add a dark color on a white / smokey white font so people are drawn to it. But it's just a little blurb of text and a few extra links, so I think having (the background) more subtly blend into the site makes sense for now.

Reply:
Another suggestion is to thin it down just a bit, removing the one massive GitHub link (and we can put it somewhere else) so the footer is a bit cleaner:

![image](https://user-images.githubusercontent.com/814322/59683636-898f9d80-91a6-11e9-85cb-b0e8794d5bb4.png)

Reply:
That might work a little better with a dark grey, if you still want to go dark.

![image](https://user-images.githubusercontent.com/814322/59683789-cc517580-91a6-11e9-8882-6eec5c0863b7.png)

Reply:
Let me know if you want to see anything else!

Reply:
I actually like either of the two border options!  I think the border gives a nice continuity without being so starkly contrasting with the white.  But I'm good with majority rules on all these design options too...

>  @cosden could you describe the accessibility issues you are worried about?

I was worried about the white on maroon in the header.  I put in #FFFFFF and #500e3d in https://webaim.org/resources/contrastchecker/ and it passed all the tests they have there.  So as far as I can tell, it's fine!

Reply:
We are almost there! @cferenba it looks like @cosden likes the border options. If you have preference for one of those, that would be majority and I can make the changes.